rust/core-lib/src/plugins/manifest.rs

```diff
@@ -96,6 +96,19 @@ pub enum PluginActivation {
 }
 
 #[cfg(not(target_os = "fuchsia"))]
```

Comment:
i'm curious about this flag, `PluginScope` is referenced in `PluginDescription`, which _isn't_ behind a gate. Am I missing something?